text/0032-minimal-relay.md

```diff
@@ -0,0 +1,190 @@
+
+However, state transitions on the Relay Chain need to be executed by _all_ validators. If any of
+those state transitions can occur on parachains, then the resources of the complement of a single
+backing group could be used to offer more cores. As in, they could be offering more coretime (a.k.a.
+blockspace) to the network.
+
+By minimising state transition logic on the Relay Chain by migrating it into "system chains" -- a
+set of parachains that, with the Relay Chain, make up the Polkadot protocol -- the Polkadot
+Ubiquitous Computer can maximise its primary offering: secure blockspace.
+
+## Stakeholders
+
+- Parachains that interact with affected logic on the Relay Chain;
+- Core protocol and XCM format developers;
+- Tooling, block explorer, and UI developers.
+
+## Explanation
+
+The following pallets and subsystems are good candidates to migrate from the Relay Chain:
+
+- Identity
+- Preimage
+- Balances
+- Staking
+	- Staking
+	- Election Provider
+	- Bags List
+	- NIS
+	- Nomination Pools
+	- Fast Unstake
+- Governance
+	- Treasury and Bounties
+	- Conviction Voting
+	- Referenda
+
+Note: The Auctions and Crowdloan pallets will be replaced by Coretime, its system chain and
+interface described in RFC-1 and RFC-5, respectively.
+
+### Migrations
+
+Some subsystems are simpler to move than others. For example, migrating Identity can be done by
+simply preventing state changes in the Relay Chain, using the Identity-related state as the genesis
+for a new chain, and launching that new chain with the genesis and logic (pallet) needed.
+
+Other subsystems cannot experience any downtime like this because they are essential to the
+network's functioning, like Staking and Governance. However, these can likely coexist with a
+similarly-permissioned system chain for some time, much like how "Gov1" and "OpenGov" coexisted at
+the latter's introduction.
+
+Specific migration plans will be included in release notes of runtimes from the Polkadot Fellowship
+when beginning the work of migrating a particular subsystem.
+
+### Interfaces
+
+The Relay Chain, in many cases, will still need to interact with these subsystems, especially
+Staking and Governance. These subsystems will require making some APIs available either via
+dispatchable calls accessible to XCM `Transact` or possibly XCM `Instruction`s in future versions.
+
+For example, Staking provides a pallet-API to register points (e.g. for block production) and
+offences (e.g. equivocation). With Staking in a system chain, that chain would need to allow the
+Relay Chain to update validator points periodically so that it can correctly calculate rewards.
+
+A pub-sub protocol may also lend itself to these types of interactions.
+
+### Deployment
+
+Actual migrations should happen based on some prioritization. This RFC proposes to migrate Identity,
+Staking, and Governance as the systems to work on first. A brief discussion on the factors involved
+in each one:
+
+#### Identity
```

Comment:
to confirm, we will have a system parachain just for identity?

Reply:
have you considered to merge identity for Kusama and Polkadot? like we have a single fellowship for both networks. I see no reason to have two set of identities on each networks.

Reply:
This is a reasonable idea. The parachain is for Identity to get it off the Relay Chain. Of course with more advanced Coretime scheduling it's probably reasonable to have slower scheduling.

The reason to keep them separate for now is that Kusama is a nice dress rehearsal for Polkadot. But I have nothing against a second migration that deprecates the Kusama chain and does all identities through Polkadot's system.